src/DataStructures/DataVector.hpp

```diff
@@ -95,266 +95,360 @@ class DataVector {
   /// \endcond
 
   /// Number of values stored
-  size_t size() const { return size_; }
+  size_t size() const noexcept { return size_; }
 
-  // @{
+  // @noexcept {
```

Comment:
is this a find/replace oops (repeated multiple times)?

Reply:
RIP :(